contentcuration/contentcuration/serializers.py

```diff
@@ -703,32 +698,6 @@ class Meta:
                   'freeze_authoring_data', 'publishing', 'original_source_node_id', 'role_visibility', 'provider', 'aggregator')
 
 
-class ContentNodeCompleteSerializer(ContentNodeEditSerializer):
```

Comment:
I'm guessing the tests for these were already removed? I was surprised to see all this serializer code being removed with no test changes.

Reply:
They were no longer referenced anywhere, no.